src/Microsoft.AspNet.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -302,6 +313,56 @@ private string GetRelativePath(RouteTemplate parsedTemplate)
             return string.Join("/", segments);
         }
 
+        private IReadOnlyList<ApiRequestFormat> GetRequestFormats(
+            ControllerActionDescriptor action,
+            IApiRequestMetadataProvider[] requestMetadataAttributes,
+            Type type)
+        {
+            var results = new List<ApiRequestFormat>();
```

Comment:
I'll move this down.

It's going to complicate the code more in both places to return null, and this isn't a perf-sensitive code path.